src/classes/HMSSDK.tsx

```diff
@@ -1111,9 +1114,9 @@ export class HMSSDK {
    * @param {*} callback
    * @memberof HMSSDK
    */
-  addEventListener = (
-    action: HMSUpdateListenerActions | HMSPIPListenerActions,
-    callback: any
+  addEventListener = <T extends HMSUpdateListenerActions | HMSPIPListenerActions>(
+    action: T | HMSPIPListenerActions,
```

Comment:
remove `HMSPIPListenerActions` from action type

```suggestion
    action: T,
```